database/migration/migrateData.ts

```diff
@@ -409,6 +409,8 @@ const migrateData = async () => {
       ADD COLUMN IF NOT EXISTS is_internal_reference_doc boolean DEFAULT FALSE NOT NULL;
       ALTER TABLE file
       ADD COLUMN IF NOT EXISTS is_external_reference_doc boolean DEFAULT FALSE NOT NULL;
+      ALTER TABLE file
+      ADD COLUMN IF NOT EXISTS is_missing boolean DEFAULT FALSE NOT NULL;
```

Comment:
Good idea to try and bundle this in with existing. However, this is the migration for versions less than `v0.3.1`, and we're past that now, so it needs to go in the latest batch. 

Just put the mutation down the bottom, straight after the last one.

Reply:
I can see that the most recent conditional migration statement is for lessthan 0.4.5,

Should I make a new conditional statement now?

Reply:
No, we're still less than 0.4.5. We're on "pre-release" 0.4.5-13, which is considered a pre-cursor to "0.4.5."